### DISCUSSION TITLE
[Bug]: MATLAB unable to start on Neurodesktop using Nectar Virtual Desktop Service

### DISCUSSION THREAD

**Discussion by neuro-sevas** [neurodesk/neurodesk.github.io-hugo#443] [Bug Report]
### Version

2024-05-25

### What platform are you seeing the problem on?

Nectar

### What happened

I clicked through 'start menu > programming > matlab > matlab 2022a' on a new virtual desktop via Nectar but the matlab activation client did not run/matlab cannot start. A terminal opens, tries to start the process of launching and then closes with fatal error. I also tried clicking on '... > matlabGUI 2022a' and it isn't able to launch, but does not display any error.

### How to reproduce

Create a new Neurodesktop via the Nectar Virtual Desktop Service (I used monash-02 as the zone) and then click through to matlab in the start menu. Below is a screenshot of the terminal right before it closes.

![Screenshot 2024-09-10 163547](https://github.com/user-attachments/assets/5efb2ce1-6201-410c-b4fa-b6123d256f6e)

**Comment by stebo85** [comment]
Dear @neuro-sevas,

I can reproduce the issue and I am looking into it now.

**Comment by stebo85** [comment]
problem found and fix is being applied. I let you know once it is live

**Comment by stebo85** [comment]
@neuro-sevas - this is completed now. Can you test again please?

**Reply by neuro-sevas** [reply]
Hi @stebo85 , it works now, thanks so much!! :)
